libhandy: 1.6.2 -> 1.6.3

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for libhandy is: "Building blocks for modern adaptive GNOME apps"

meta.homepage for libhandy is: "https://gitlab.gnome.org/GNOME/libhandy"

meta.changelog for libhandy is: "https://gitlab.gnome.org/GNOME/libhandy/-/tags/1.6.3"


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes

<del>
https://gitlab.gnome.org/GNOME/libhandy/-/compare/1.6.0...1.7.0

- HdyDeck
  - Remove a non-existent property mention
- HdyStyleManager
  - Correctly reset the theme if we're not overriding it. Fixes switching away from HighContrast.
  - Fix a typo in docs
  - Remove a leftover debug message
  - Add environment variables to control styles from Builder
- Fix license in docs: *GPL-2.1-or-later → LGPL-2.1-or-later*
- Fix a Meson error when using as a subproject
</del>

https://gitlab.gnome.org/GNOME/libhandy/-/compare/1.6.2...1.6.3


- Remove a non-existent property mention from HdyDeck docs
- Remove a leftover debug message from HdySettings


###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 1.7.0 with grep in /nix/store/hi0jsskgsbfd3qgvz4bwvg5mh84lqdqd-libhandy-1.7.0
- found 1.7.0 in filename of file in /nix/store/hi0jsskgsbfd3qgvz4bwvg5mh84lqdqd-libhandy-1.7.0

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
153 total rebuild path(s)

135 package rebuild(s)

135 x86_64-linux rebuild(s)
18 x86_64-darwin rebuild(s)


First fifty rebuilds by attrpath
apostrophe
bottles
caerbannog
calls
cambalache
chatty
cobang
cozy
cpupower-gui
denemo
dropbox-cli
eiciel
elementary-planner
empathy
epiphany
evince
fondo
glom
gnome-builder
gnome-connections
gnome-console
gnome-feeds
gnome-photos
gnome-podcasts
gnome-usage
gnome.cheese
gnome.empathy
gnome.eog
gnome.epiphany
gnome.evince
gnome.file-roller
gnome.geary
gnome.gnome-books
gnome.gnome-boxes
gnome.gnome-control-center
gnome.gnome-disk-utility
gnome.gnome-documents
gnome.gnome-initial-setup
gnome.gnome-logs
gnome.gnome-maps
gnome.gnome-notes
gnome.gnome-screenshot
gnome.gnome-system-monitor
gnome.gnome-terminal
gnome.gnome-tweaks
gnome.gnome-user-share
gnome.nautilus
gnome.nautilus-python
gnome.seahorse
gnome.simple-scan
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/hi0jsskgsbfd3qgvz4bwvg5mh84lqdqd-libhandy-1.7.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A libhandy https://github.com/r-ryantm/nixpkgs/archive/3fd07d83a3399a68349ff60f75aec3bd7ee6e282.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/hi0jsskgsbfd3qgvz4bwvg5mh84lqdqd-libhandy-1.7.0
ls -la /nix/store/hi0jsskgsbfd3qgvz4bwvg5mh84lqdqd-libhandy-1.7.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

nixpkgs-review took longer than 45m and timed out

---

###### Maintainer pings

cc @bobby285271 @hedning @jtojnar @dasj19 @maxeaubrey for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

r-ryantm committed:
libhandy: 1.6.2 → 1.6.3

https://gitlab.gnome.org/GNOME/libhandy/-/compare/1.6.2...1.6.3

jtojnar reviewed (changes requested):
This is an unstable version: https://matrix.to/#/!NhkytdFqgkyapgssoN:talk.puri.sm/$RM--dDaR7foOfzTU6P1luyVG2zRynNpm2nLnz9fBL9I?via=talk.puri.sm&via=matrix.org&via=gnome.org

bobby285271 commented:
Joined the Matrix room... Just noticed there is also 1.6.3
https://gitlab.gnome.org/GNOME/libhandy/-/compare/1.6.2...1.6.3

bobby285271 committed:
libhandy: fix updateScript versionPolicy

libhandy follows the old GNOME versioning scheme.

https://matrix.to/#/!NhkytdFqgkyapgssoN:talk.puri.sm/$RM--dDaR7foOfzTU6P1luyVG2zRynNpm2nLnz9fBL9I?via=talk.puri.sm&via=matrix.org&via=gnome.org
